tests/jerry/es2015/typedArray-find.js

```diff
@@ -0,0 +1,55 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var int_array = new Int8Array ([5, 12, 8, 100, 44]);
+
+function bigger_than_10 (element) {
+  return element > 10;
+}
+
+function less_than_0 (element) {
+  return element < 0;
+}
+
+assert (int_array.find (bigger_than_10) === 12);
+assert (int_array.find (less_than_0) === undefined);
+
+function isPrime (element, index, array) {
```

Comment:
We are not using the array here. Could we have a test case which tests all the arguments?